frontend/src/types.ts

```diff
@@ -2256,6 +2256,7 @@ export type HotKey =
     | 'arrowright'
     | 'arrowdown'
     | 'arrowup'
+    | 'forwardslash'
```

Comment:
The problem with this is that this hotkey isn't going to actually work, as the `event.key` property takes the value of "/" for the forward slash (this would only work if it were "ForwardSlash" – "ArrowDown"-style – but that's not the case)